cmd/infra/powervs/validate.go

```diff
@@ -131,10 +131,40 @@ func validateVpc(vpcName string, resourceGroupID string, v1 *vpcv1.VpcV1) (vpc *
 		return
 	}
 
+	if vpc == nil {
+		err = fmt.Errorf("%s vpc not found", vpcName)
+		return
+	}
+
 	if vpc != nil {
```

Comment:
why do we need this `if vpc != nil` block? post above nil check it will always be non nil here!

Reply:
yes sorry, removed the nil validation.